egs/wsj/s5/local/nnet3/run_blstm.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
```

Comment:
I think you should eliminate this file, as we could just use lstm_delay string with the usual run_lstm.sh to specify blstms.